src/components/post.vue

```diff
@@ -19,13 +31,28 @@ export default defineComponent({
         required: true
       }
     },
-    setup() {
-      const formatDate = (date: string): string => {
-        return moment(String(date)).format('MMMM DD, YYYY')
+    setup(props) {
+      const formattedDate = (date: string) => {
```

Comment:
you can just use `props.post.date` instead of adding a parameter. Also, you can change this into a `computed` variable.